src/containers/PatientProfilePage.tsx

```diff
@@ -154,6 +162,25 @@ const PatientProfilePage = ({
     history.replace('/');
   };
 
+  const handleDeleteClick = () => {
+    setDeleteClicked(true);
+  };
+
+  const handleDeleteCancel = () => {
+    setDeleteClicked(false);
+  };
+
+  const handldeDeleteConfirm = () => {
+    if (mode === 'edit') {
+      deletePatient({
+        variables: {
+          id: patientId,
+        },
+      });
+    }
+    history.replace('/');
```

Comment:
how come we're redirecting to the home page here? also would `history.push` be more appropriate?

Reply:
I'm not totally sure where we should be redirecting to. As for using `history.push`: it doesn't make sense to have the deleted patient in our history. Ideally, the user should not have a method for coming back to that patient.

Reply:
let's redirect back to the CCP dashboard for now, we can revisit this if product mentions otherwise